hikari/api/rest.py

```diff
@@ -6917,6 +6925,17 @@ async def create_slash_command(
 
             Defaults to `builtins.True`.
 
+            .. deprecated:: 2.0.0.dev109
+                Use `default_member_permissions` instead.
+        default_member_permissions : hikari.undefined.UndefinedOr[hikari.permissions.Permissions]
+            Member permissions necessary to utilize this command by default.
+
+            If `0`, then it will be disabled by default.
```

Comment:
As outlined by the Discord API docs, the commands by default are disabled except for administrators if set to `0`.

Reply:
These are defaults. They can of course be overwritten per server. But I dont think we need to document that specifically, we already clarify these are default values that can be modified

Reply:
It may be a good idea to point out specifically still, as some users may incorrectly believe that this, for example, may allow for the creation of developer-only slash commands.

Reply:
> It may be a good idea to point out specifically still, as some users may incorrectly believe that this, for example, may allow for the creation of developer-only slash commands.

Discord also updated their documentation later to point this out as well (though it really should have done that in the first place). I still recommend to do the same here for clarity purposes.